Application/EdFi.Ods.AdminApi/EdFi.Ods.AdminApi.csproj

```diff
@@ -1,49 +1,50 @@
-  </PropertyGroup>
-  <ItemGroup>
-    <Content Include="EdFi.Ods.AdminApi.nuspec" CopyToPublishDirectory="Always" CopyToOutputDirectory="Never" />
-    <Content Include="Artifacts\**" CopyToPublishDirectory="Always" CopyToOutputDirectory="Never" />
-    <Content Include="Schema\Ed-Fi-ODS-API-Profile.xsd" CopyToPublishDirectory="Always" CopyToOutputDirectory="Always" />
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="AspNetCore.HealthChecks.NpgSql" Version="8.0.1" />
-    <PackageReference Include="AspNetCore.HealthChecks.SqlServer" Version="8.0.2" />
-    <PackageReference Include="Asp.Versioning.Http" Version="8.1.0" />
-    <PackageReference Include="AspNetCoreRateLimit" Version="5.0.0" />
-    <PackageReference Include="AutoMapper" Version="13.0.1" />
-    <PackageReference Include="Azure.Identity" Version="1.12.0" />
-    <PackageReference Include="EdFi.Suite3.Admin.DataAccess" Version="7.3.67" />
-    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="7.3.431" />
-    <PackageReference Include="FluentValidation.AspNetCore" Version="11.3.0" />
-    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="8.0.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Log4Net.AspNetCore" Version="8.0.0" />
-    <PackageReference Include="Microsoft.Identity.Client" Version="4.63.0" />
-    <PackageReference Include="NJsonSchema" Version="11.0.2" />
-    <PackageReference Include="OpenIddict.AspNetCore" Version="4.10.1" />
-    <PackageReference Include="OpenIddict.EntityFrameworkCore" Version="4.10.1" />
-    <PackageReference Include="SonarAnalyzer.CSharp" Version="9.24.0.89429">
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.7.0" />
-    <PackageReference Include="Swashbuckle.AspNetCore.Annotations" Version="6.7.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="8.0.8" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Abstractions" Version="8.0.8" />
-  </ItemGroup>
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <IsPackable>true</IsPackable>
+    <NoWarn>NU5100, NU5124</NoWarn>
+    <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+    <DockerfileContext>../..</DockerfileContext>
+    <DockerfileFile>dev.pgsql.Dockerfile</DockerfileFile>
+    <DockerfileBuildArguments>--no-cache</DockerfileBuildArguments>
+    <DockerfileRunEnvironmentFiles>.env</DockerfileRunEnvironmentFiles>
+    <DockerfileTag>adminapi-dev</DockerfileTag>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <UserSecretsId>a39ca29f-5ebf-412e-bca9-53fc7221286b</UserSecretsId>

```

Comment:
It looks like the line endings changed. Line 15 above is the real change: this enables creating a user secrets file that will override `appsettings.json` .